catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -142,6 +142,10 @@ public CreateStorageResponse create(CreateStorageRequest createRequest)
     public ReadStorageResponse read(ReadStorageRequest readRequest) throws StorageException {
         LOGGER.trace("ENTERING: read");
 
+        if (readRequest.getResourceUri() == null) {
+            return new ReadStorageResponseImpl(readRequest);
```

Comment:
Whats funny is in my catalog UI code (in the other fork) i checked and somehow I'm already checking if this is null... probably because I don't trust our API's. But hoping everyone else did the same 😄 
aka, I will investigate to make sure none of that nonsense is occuring in here or "in the wild".